keras_cv/metrics/f_scores.py

```diff
@@ -0,0 +1,356 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+import tensorflow.keras as keras
+
+
+class FBetaScore(keras.metrics.Metric):
+    r"""Computes F-Beta score.
+
+    It is the weighted harmonic mean of precision
+    and recall. Output range is `[0, 1]`. Works for
+    both multi-class and multi-label classification.
+    
+    $$
+    F_{\beta} = (1 + \beta^2) * \frac{\textrm{precision} * \textrm{recall}}{(\beta^2 \cdot \textrm{precision}) + \textrm{recall}}
+    $$
+    
+    Args:
+        num_classes: Number of unique classes in the dataset.
+        average: Type of averaging to be performed on data.
+            Acceptable values are `None`, `micro`, `macro` and
+            `weighted`. Default value is None.
+        beta: Determines the weight of precision and recall
```

Comment:
how does this parameter work?  Are numbers > 1 giving more weights to recall?

Reply:
I think yes.